src/generate.ts

```diff
@@ -41,13 +42,11 @@ async function processConfig(config: Configuration): Promise<Output[]> {
     visitorConfig["$filename"] = file;
 
     log.debug(
-      `Generating source for '${file}' with generator from ${url} with config\n${
-        JSON.stringify(
-          visitorConfig,
-          null,
-          2,
-        )
-      }`,
+      `Generating source for '${file}' with generator from ${url} with config\n${JSON.stringify(
```

Comment:
I'm using `deno fmt src` to format all the code instead of using prettier. It looks like the formatter you are using is making a change here. Could you pls run `deno fmt src`?